etc/dbus-serialbattery/dbushelper.py

```diff
@@ -1100,4 +1119,18 @@ def saveBatteryOptions(self) -> bool:
                 + f"after {self.battery.soc_reset_last_reached}",
             )
 
+        if abs(self.battery.soc_calc - self.save_charge_details_last["soc_calc"]) > 1.0:
+            self.save_charge_details_last["soc_calc"] = self.battery.soc_calc
+            result = result and self.setSetting(
+                get_bus(),
+                "com.victronenergy.settings",
+                self.path_battery,
+                "SocCalculated",
+                self.battery.soc_calc,
+            )
+            logger.info(
```

Comment:
I added it as info, so see, if the new changes are working, since my added values do not change very often. You can already change it to debug, if it floods your log.

Reply:
I changed it to debug.